modules/react/common/stories/BasicCompoundComponent.stories.mdx

```diff
@@ -0,0 +1,268 @@
+import {Meta} from '@storybook/addon-docs';
```

Comment:
should probably rename this file to something like `CreateComponentUtilities`

Reply:
Agreed, this file should be renamed. Another option would be `CompoundComponentUtilities.stories.mdx`